File: src/model/Wallet.java
package model;

import java.util.ArrayList;
import java.util.List;

public class Wallet {
    private Money money;
    private List<Document> documents;

    public Wallet() {
        this.money = new Money();
        this.documents = new ArrayList<>();
    }

    public Money getMoney() {
        return money;
    }

    public void setMoney(Money money) {
        this.money = money;
    }

    public List<Document> getDocuments() {
        return documents;
    }

    public void setDocuments(List<Document> documents) {
        this.documents = documents;
    }
}

File: src/model/wallet.py
from src.model.money import Money


class Wallet:
    def __init__(self):
        self.__money: Money = Money()

    @property
    def money(self) -> Money:
        return self.__money

    @money.setter
    def money(self, money: Money) -> None:
        self.__money = money
